Gems/AtomLyIntegration/CommonFeatures/Code/Source/ReflectionProbe/EditorReflectionProbeComponent.cpp

```diff
@@ -47,7 +47,7 @@ namespace AZ
                     editContext->Class<EditorReflectionProbeComponent>(
                         "Reflection Probe", "The ReflectionProbe component captures an IBL specular reflection at a specific position in the level")
                         ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
-                            ->Attribute(AZ::Edit::Attributes::Category, "Atom")
```

Comment:
nit: I'd defer to Doug/Jonny on this, but to me 'Lights' implies actual light types. I wonder if 'Lighting' would make more sense as a category than 'Lights' to include relfection probes, diffuse GI, etc.

Reply:
I was thinking the same. Will change to `Lighting`